commands/hugo.go

```diff
@@ -211,7 +211,7 @@ func initRootPersistentFlags() {
 
 	// Set bash-completion
 	validConfigFilenames := []string{"json", "js", "yaml", "yml", "toml", "tml"}
-	HugoCmd.PersistentFlags().SetAnnotation("config", cobra.BashCompFilenameExt, validConfigFilenames)
+	_ = HugoCmd.PersistentFlags().SetAnnotation("config", cobra.BashCompFilenameExt, validConfigFilenames)
```

Comment:
It looks better without that _, but we have had a fair amount of issues related to not handling returned errors. And it signals that we know about the error, and not the least: being able to run `errcheck` and using the exit code as a signal, means we can add it to the `make check` if we want.